_source/logzio_collections/_tracing-sources/cs_open_tracing.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Sending traces from C# application
+logo:
+  logofile: dotnet.svg
+  orientation: vertical
+data-source: dotnet
+description: How to send traces from C# application
+open-source:
+  - title: 
+    github-repo: jaeger-logzio
+contributors:
+  - yyyogev
+  - yberlinger
+shipping-tags:
+  - from-your-code
+  - traces
+---
+<!-- tabContainer:start -->
+<div class="branching-container">
+
+* [Auto instrumentation](#tab1)
+* [From your code](#tab2)
+{:.branching-tabs}
+
+<!-- tab:start -->
+<div id="tab1">
+
+# OpenTracing Instrumentation for C\#
+
+This article describes how to automatically extract Traces from your C# based app with OpenTracing instrumentation.
+
+## OpenTracing and Jaeger
+
+OpenTracing is an API standard of instrumentation for distributed tracing.
+Jaeger is one implementation of that standard and our most recommended practice for C#. It has a set of libraries and tools to provide open source implementation 
+for distributed tracing.
+
+## OpenTracing for C\#
+
+### Requirements
+Add the following packages to your solution:
```

Comment:
@yyyogev - where is user supposed to get these packages from?

Reply:
There are several ways, the most common way is via NuGet

Reply:
@yyyogev  -Apologies. My intent was not clear:

 Is it reasonable for us to assume that users will know where to get the packages from or should we give them explicit guidance on where to obtain the packages?

Reply:
Nah, it's a known knowledge